libs/ipfs-http-client/Makefile

```diff
@@ -0,0 +1,74 @@
+# 
+# This is free software, licensed under the GNU General Public License v2.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=ipfs-http-client
+PKG_RELEASE:=1
+
+PKG_MAINTAINER:=Leonid Esman <leonid.esman@gmail.com>
+PKG_LICENSE:=MIT
+PKG_LICENSE_FILES:=LICENSE.MIT
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/vasild/cpp-ipfs-http-client.git
+#PKG_SOURCE_DATE:=2019-11-05
+PKG_SOURCE_VERSION:=763e59ad698f3e3846f85df11e01c18ef3fbc401
+PKG_MIRROR_HASH:=39b7118452c33782f0a370fce1e9b8e1b75dfe2f75bbf4c850d6886685d82a3e
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/$(PKG_NAME)/Default/description
+	IPFS (the InterPlanetary File System) is the Distributed Web. 
+	Specs, docs, sources, links: https://ipfs.io/ and https://github.com/ipfs.
+	This is Vasil Dimov's C++ IPFS HTTP API client library.
+endef
+
+define Package/lib$(PKG_NAME)
+	SECTION:=libs
+	CATEGORY:=Libraries
+	TITLE:=IPFS client library
+	URL:=https://github.com/vasild/cpp-ipfs-http-client
+	DEPENDS:= +libcurl +libstdcpp
+	PKG_BUILD_DEPENDS:=nlohmannjson
+endef
+
+define Package/lib$(PKG_NAME)/description
+	$(call Package/$(PKG_NAME)/Default/description)
+	This package contains shared library.
+endef
+
+define Package/$(PKG_NAME)-tests
+	SECTION:=utils
+	CATEGORY:=Utilities
+	TITLE:=IPFS client library tests
+	URL:=https://github.com/vasild/cpp-ipfs-http-client
+	DEPENDS:=+libipfs-http-client +libcurl
```

Comment:
are these tests actualyl something someone would ever want on their target device?

Reply:
It's optional